sympy/sets/sets.py

```diff
@@ -1628,6 +1628,10 @@ class FiniteSet(Set, EvalfMixin):
     >>> 3 in FiniteSet(1, 2, 3, 4)
     True
 
+    >>> members = [1, 2, 3, 4]
+    >>> FiniteSet(*members)
```

Comment:
> Sure, the point is not to teach Python

But you suggest exactly this.  Even Python's tutorial has [examples of this unpacking](https://docs.python.org/3/tutorial/controlflow.html#unpacking-argument-lists).